docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
       MAILNAME: developmentseed.org
       DISABLE_IPV6: 1
 
-  opensearch:
+  opensearch-node-1:
```

Comment:
We will never have multiple nodes running for local development which is why I removed `node-1` - is there any reason to add it back?

Reply:
No. I did not notice that change. I'll revert.